nexus_constructor/qml_models/pixel_models.py

```diff
@@ -160,7 +140,7 @@ def get_pixel_model(self):
     @Slot(int, "QVariant")
     def set_pixel_model(self, index, instrument: InstrumentModel):
         component = instrument.components[index]
-        if isinstance(component.pixel_data, PixelMapping):
+        if component.pixel_data.type == "PixelMapping":
```

Comment:
@DominicOram Jack says switching from using `isinstance` to having a string type field is a change you suggested. Is that right? Why do you prefer that?

Reply:
I don't remember suggesting that. Sorry @rerpha, I probably hadn't had enough coffee, `isinstance` is better.